code/datums/antagonists/datum_clockcult.dm

```diff
@@ -18,6 +18,14 @@
 		. = is_eligible_servant(new_body)
 
 /datum/antagonist/clockcultist/give_to_body(mob/living/new_body)
+	if(iscyborg(new_body))
+		var/mob/living/silicon/robot/R = new_body
+		if(R.deployed)
+			var/mob/living/silicon/ai/AI = R.mainframe
+			R.undeploy()
+			var/converted = add_servant_of_ratvar(AI, silent_update)
+			AI << "<span class='userdanger'>Anomaly Detected. Returned to core!</span>"	//The AI needs to be in its core to properly be converted
```

Comment:
Apologies, I forgot to change that portion of the code. I will catch it in my next commit.